src/holofuel/pages/Home/Home.js

```diff
@@ -62,47 +64,50 @@ export default function Home () {
   }, [filterActionableTransactionsByStatus, newMessage])
 
   return <PrimaryLayout headerProps={{ title: 'Home' }}>
-    <div styleName='avatar'>
-      <CopyAgentId agent={holofuelUser} isMe>
-        <HashAvatar seed={holofuelUser.id} size={48} />
-      </CopyAgentId>
-    </div>
-    <div styleName='greeting'>{greeting}</div>
-    <div styleName='button-row'>
-      <Button onClick={goToOfferRequest} styleName='send-button'>Send / Request</Button>
-    </div>
-
-    <div styleName='balance-and-transactions'>
-      <Link to={HISTORY_PATH} styleName='balance-link'>
-        <div styleName='balance'>
-          <div styleName='balance-header'>
-            <div styleName='balance-label'>
-              Available Balance
-            </div>
-            <div styleName='balance-arrow'>
-              >
+    <div styleName='container'>
```

Comment:
Let's add the `showAlphaFlag` prop here.

```
<PrimaryLayout headerProps={{ title: 'Home' }} showAlphaFlag={false}>
```

Reply:
The designs have the Alpha Flag on the Home page.

https://www.figma.com/file/CCt10PCyyCVeo6G6T4j28O/HoloFuel?node-id=0%3A3681